src/player.cpp

```diff
@@ -229,7 +229,10 @@ void Player::clearHud()
 
 RemotePlayer::RemotePlayer(IGameDef *gamedef, const char *name):
 	Player(gamedef, name),
-	m_sao(NULL)
+	m_sao(NULL),
+	m_last_chat_message_sent(time(NULL)),
```

Comment:
TimeGetter is only available in clientlauncher.h, and i think we don't need this too generic way